stdlib/public/TensorFlow/Dataset.swift

```diff
@@ -46,6 +46,8 @@ public extension SingleValueDataset {
   // users must specify the element shape for today.
   @inlinable @inline(__always)
   public init(elements: Tensor<Scalar>, elementShape: TensorShape) {
+    // A dataset graph function must run on TF CPU.
+    TensorFlow.enableCPU()
```

Comment:
There's no need to say `TensorFlow.`. `enableXXX()` are just global functions.

Reply:
Good catch. Removed.